libs/langchain/langchain/document_loaders/parsers/audio.py

```diff
@@ -136,10 +156,19 @@ def __init__(self, device: str = "0", lang_model: Optional[str] = None):
         # load model for inference
         self.pipe = pipeline(
             "automatic-speech-recognition",
-            model="openai/whisper-medium",
+            model=self.lang_model,  # fix to use model name that was evaluated earlier
             chunk_length_s=30,
             device=self.device,
         )
+        try:
+            if forced_decoder_ids is not None:
```

Comment:
nit: i feel like its slightly nicer to do:
```
if ...:
    try:
    ...
    except:
    ...
```
dont care super strongly tho

Reply:
Thank you. Fixed.